src/backend/tests/bindings/typedef-anon-struct.sexp

```diff
@@ -0,0 +1,39 @@
+(c-lisp
+    ,@(include
+        (shape.h)
+        ()
```

Comment:
I don't like this syntax that much because of empty lists and non named stuff

Reply:
Right. I'm open to writing a new macro that wraps this macro, takes no arguments and includes all available functions + structs they depend on.
Available functions can be scraped from the JSON AST.

Reply:
Although the ability to have unspecified arguments is an important capability. So maybe we could implement keyword arguments in Prelisp?
Perhaps something like this, in sexp:
```lisp
,(macro-fn parg1 (parg2_1 parg2_2) ((karg1 . val1) (karg2 . val2)))
```
and equivalent Python call would be
```python
macro_fn("parg1", ["parg2_1" "parg2-2"], karg1="val1", karg2="val2")
```

Reply:
Alright, Guile supports keyword arguments, so we can use that.
```scheme
; Procedure definition
(define* (func #:key arg)
    arg)

; Procedure call
(display (func #:arg 'value))
```
result:
```
value
```

Although there are a lot of Scheme implementations (including Guile) that support some form of keyword arguments, the Scheme spec doesn't provide keyword arguments.

Reply:
try reading the this using (read) first and then see how it works. It's possible it's just a symbol?